### DISCUSSION TITLE
Deploying Aspire apps to files not to azure

### DISCUSSION THREAD

**Discussion by zandiarash** [microsoft/aspire#5590] [Q&A] [answered]
Is there a way to publish Aspire apps to file or something other than Azure ?

**Comment by davidfowl** [comment]
No there’s no deploy to files. You can however continue publishing however you were doing it before aspire.

**Reply by zandiarash** [reply]
Is there any plan to use Aspire for deployment?

**Reply by davidfowl** [reply, marked as answer]
There's [aspirate](https://github.com/prom3theu5/aspirational-manifests) which supports emitting a docker composed based or Kubernetes based deployment.

**Reply by zandiarash** [reply]
Do you mean with Aspir8 I can deploy to docker or k8s ?

**Reply by davidfowl** [reply]
Correct.

**Reply by zandiarash** [reply]
Thanks Bro❤️
